main.go

```diff
@@ -35,8 +39,21 @@ var (
 	caPath   = flag.String("ca", "ca.pem", "Path to CA to auth clients against")
 	noverify = flag.Bool("no-verify", false, "Disable verification, (voids the entire point of this, just for testing)")
 	regexStr = flag.String("regex", ".*", "Regular expression to match against CNs (start and end anchors will be added)")
+	gphtStr  = flag.String("graphite", "", "Graphite endpoint")
 )
 
+func runMetrics() {
+	exp.Exp(metrics.DefaultRegistry)
+	if *gphtStr != "" {
+		addr, err := net.ResolveTCPAddr("tcp", *gphtStr)
+		if err != nil {
+			log.Fatal("could not resolve graphite address, ", err.Error())
+		}
+		go graphite.Graphite(metrics.DefaultRegistry,
+			1*time.Second, "tlsproxy", addr)
```

Comment:
Assuming `tlsproxy` is the prefix here, it'd be great to be able to configure it (i.e. we'd want `tlsproxy.harmony` vs `tlsproxy.<whatever else we decide to use`)